README.md

```diff
@@ -21,10 +21,10 @@ configurations can be used to configure other properties of the ECS Agent contai
 
 Additionally, the following environment variable(s) can be used to configure the behavior of the RPM:
 
-| Environment Variable Name | Example Value(s)            | Description | Default value |
-|:----------------|:----------------------------|:------------|:-----------------------|
-| `ECS_SKIP_LOCALHOST_TRAFFIC_FILTER` | &lt;true &#124; false&gt; | By default, the ecs-init service adds an iptable rule to drop non-local packets to localhost if they're not part of an existing forwarded connection or DNAT, and removes the rule upon stop. If `ECS_SKIP_LOCALHOST_TRAFFIC_FILTER` is set to true, this rule will not be added/removed. | false |
-| `ECS_ALLOW_OFFHOST_INTROSPECTION_ACCESS` | &lt;true &#124; false&gt; | By default, the ecs-init service adds an iptable rule to block access to ECS Agent's introspection port from off-host (or containers in awsvpc network mode), and removes the rule upon stop. If `ECS_ALLOW_OFFHOST_INTROSPECTION_ACCESS` is set to true, this rule will not be added/removed. | false |
+| Environment Variable Name | Example Value(s)            | Description | Default value | On Amazon Linux 1 | On Amazon Linux 2 |
+|:----------------|:----------------------------|:------------|:-----------------------|:------------|:------------------|
+| `ECS_SKIP_LOCALHOST_TRAFFIC_FILTER` | &lt;true &#124; false&gt; | By default, the ecs-init service adds an iptable rule to drop non-local packets to localhost if they're not part of an existing forwarded connection or DNAT, and removes the rule upon stop. If `ECS_SKIP_LOCALHOST_TRAFFIC_FILTER` is set to true, this rule will not be added/removed. | false | The flag can be turned on via adding `env ECS_SKIP_LOCALHOST_TRAFFIC_FILTER=true` to /etc/init/ecs.conf | The flag can be turned on via adding `ECS_SKIP_LOCALHOST_TRAFFIC_FILTER=true` to /etc/ecs/ecs.config
```

Comment:
every env can be set in the same manner, so adding a column explaining each seems redundant. probably better to explain the usage below this table? and just use one env as example